compute/instances/api-cli/migrating-gp-to-pro.mdx

```diff
@@ -172,21 +171,21 @@ Local and block snapshots have different formats. To support the migration from
     --header 'X-Auth-Token: '$SCW_SECRET_KEY'' \
     --data-raw '{
       "action": "poweron"
-    }'
+  }'
```

Comment:
The bracket should be aligned with --data-raw
The problem is with the curl command above